### DISCUSSION TITLE
翻译问题

### DISCUSSION THREAD

**Discussion by Gaozx1** [beijixiaohu/OJBetter#194] [General]
Markdown格式问题
如:
你有两个盛水的容器。第一个容器含有 a
 克水,第二个容器含有 b
 克水。两个容器都非常大，可以容纳任何数量的水。

你也有一个空杯子，可以容纳***** c
 克的水。

在一个动作,你可以挖到 c
 克水从任何船和其他倒到** **。注意，在一次移动中倾倒的水的质量不一定是一个整数。

要使容器里的水的质量相等，最少要走多少步？请注意，除了所描述的移动之外，您不能执行任何操作。

多个*号

**Comment by beijixiaohu** [comment]
题目链接发下，我去看看，一般都是翻译服务的问题，不是脚本的问题
